release-notes/Cloud/dataminer_services_change_log_2025.md

```diff
@@ -9,6 +9,12 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 28 April 2025 - Enhancement - Usage API - Performance and stability improvements [ID 42791]
+
+It was possible that an export request failed due to OOM, this has been fixed.
```

Comment:
Could you clarify what you mean with OOM?

Reply:
out of memory